Loop/Window Management/WindowDirection.swift

```diff
@@ -54,7 +55,7 @@ enum WindowDirection: String, CaseIterable, Identifiable, Codable {
 
     // These are used in the menubar resize submenu & keybind configuratio
     static var general: [WindowDirection] { [.fullscreen, .maximize, .almostMaximize, .center, .macOSCenter, .minimize, .hide] }
-    static var halves: [WindowDirection] { [.topHalf, .bottomHalf, .leftHalf, .rightHalf] }
+    static var halves: [WindowDirection] { [.topHalf, .bottomHalf, .leftHalf, .rightHalf, .horizontalCenterHalf, .verticalCenterHalf] }
```

Comment:
I would suggest ordering it so that it's:
1. Top Half
2. Vertical Center Half
3. Bottom Half
4. Left Half
5. Horizontal Center Half
6. Right Half

so that halves are ordered in the same pattern as thirds :)
Other than that, this is looking great!